admin/create-dumps.sh

```diff
@@ -32,14 +32,6 @@ function add_rsync_include_rule {
     echo "$SHA256_FILE_RULE" >> "$RULE_FILE"
 }
 
-if [[ "${CONTAINER_NAME}" = "listenbrainz-cron-prod" && "${PROD}" = "prod" ]]
```

Comment:
Sorry for the unrelated fix here -- I think this will be replaced by another PR possibly before this is even merged.